.github/workflows/push-tests.yml.j2

```diff
@@ -1,10 +1,10 @@
-{% if distro_release == "rawhide" %}
```

Comment:
Why don't we completely remove this template file. It's not used now.

Reply:
@adamkankovsky he did not mean to remote the workflow, he meant to remove the template file. The actual workflow can stay. Please do this in another PR, in order to keep this moving.